src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    String ballColor;
+    int ballNumber;
+
```

Comment:
According to encapsulation principle variables should be of private access, while the methods are public. So use private access modifier for variables